monitoring/prober/rid/test_token_validation.py

```diff
@@ -47,7 +44,7 @@ def test_put_isa_with_read_only_scope_token(rogue_session, session, isa2_uuid):
               'time_end': time_end.strftime(common.DATE_FORMAT),
           },
           'flights_url': 'https://example.com/dss',
-      })
+      }, scope='dss.read.identification_service_areas')
```

Comment:
nit: Seems like this could use a constant, too.

Reply:
Done.